use-cases/vehicle-routing-capacity/src/main/java/org/acme/vehiclerouting/rest/VehicleRoutePlanResource.java

```diff
@@ -62,9 +62,9 @@ public String solve(VehicleRoutePlan problem) {
                 jobId_ -> jobIdToJob.get(jobId).routePlan,
                 solution -> jobIdToJob.put(jobId, Job.newRoutePlan(solution)),
                 (jobId_, exception) -> {
-            jobIdToJob.put(jobId, Job.error(jobId_, exception));
-            LOGGER.error("Failed solving jobId ({}).", jobId,  exception);}
-        );
+                    jobIdToJob.put(jobId, Job.error(jobId_, exception));
+                    LOGGER.error("Failed solving jobId ({}).", jobId, exception);
+                });
```

Comment:
How did this escape auto-formatting in CI?

Reply:
Because there is no auto-formatting in QS. No build-parent from the solver repo, no auto-formatting.